package.json

```diff
@@ -31,7 +31,7 @@
     }
   },
   "devDependencies": {
-    "@babel/plugin-transform-modules-commonjs": "^7.16.5",
+    "@babel/plugin-transform-modules-commonjs": "^7.22.9",
```

Comment:
Should there be a corresponding yarn.lock file change?

Reply:
Thanks for the reminder! Updated.